src/main/webapp/app/pages/curation/header/MutationsSectionHeader.tsx

```diff
@@ -342,6 +348,33 @@ function MutationsSectionHeader({
                 );
               })}
             </Row>
+            <h6 className="mb-2 mt-2">Reviewed</h6>
+            <Row className="align-items-start justify-content-start">
+              {tempReviewedFilter.map((filter, index) => {
+                const isDisabled = !checkboxEnabled(FilterType.REVIEWED, filter.label);
+
+                return (
+                  <Col className="col-2" key={filter.label}>
+                    <InputGroup>
+                      <Input
+                        id={`tx-level-filter-${filter.label}`}
+                        onChange={() => handleFilterCheckboxChange(index, setTempReviewedFilter)}
+                        checked={filter.selected}
+                        disabled={isDisabled}
+                        style={{ cursor: `${isDisabled ? null : 'pointer'}`, marginLeft: '0px' }}
+                        type="checkbox"
+                      />
+                      <Label
+                        for={`tx-level-filter-${filter.label}`}
+                        style={{ cursor: `${isDisabled ? null : 'pointer'}`, marginLeft: CHECKBOX_LABEL_LEFT_MARGIN }}
+                      >
+                        {filter.label}
+                      </Label>
+                    </InputGroup>
```

Comment:
This InputGroup seems to be replicated quite a lot. Can we make it a component?

Reply:
Oh sorry about this, but I was actually working on a different PR and I accidentally forked this one off of that. This was not supposed to make it into this PR, so I just removed it, but I will definitely make it into a component for that PR